app/Foliage/HackageSecurity.hs

```diff
@@ -32,20 +37,36 @@ computeFileInfoSimple fp = do
 
 createKeys :: FilePath -> IO ()
 createKeys base = do
+  putStrLn "  root keys:"
   createDirectoryIfMissing True (base </> "root")
   replicateM_ 3 $ createKey' KeyTypeEd25519 >>= writeKeyWithId (base </> "root")
+  putStrLn "  target keys:"
   createDirectoryIfMissing True (base </> "target")
   replicateM_ 3 $ createKey' KeyTypeEd25519 >>= writeKeyWithId (base </> "target")
+  putStrLn "  timestamp keys:"
   createDirectoryIfMissing True (base </> "timestamp")
   replicateM_ 1 $ createKey' KeyTypeEd25519 >>= writeKeyWithId (base </> "timestamp")
+  putStrLn "  snapshot keys:"
   createDirectoryIfMissing True (base </> "snapshot")
   replicateM_ 1 $ createKey' KeyTypeEd25519 >>= writeKeyWithId (base </> "snapshot")
+  putStrLn "  mirrors keys:"
   createDirectoryIfMissing True (base </> "mirrors")
   replicateM_ 3 $ createKey' KeyTypeEd25519 >>= writeKeyWithId (base </> "mirrors")
```

Comment:
```suggestion
```

I think we need to print only the root keys since the user has to copy only those in their cabal.project.

Reply:
I'm confused by this. This seems to not only remove the printing of the other keys, but also stop them being created at all? Do we not need those?

Reply:
:facepalm: It did pass through my mind when I last reviewed this that perhaps it was better to separate the creating and the printing of the keys. Then I went ahead and mixed the two things up :cry: